raiden-ts/src/raiden.ts

```diff
@@ -425,6 +448,20 @@ export class Raiden {
     return this.deps.provider.blockNumber || (await this.deps.provider.getBlockNumber());
   }
 
+  public config(newConfig: Partial<RaidenConfig>) {
+    this.deps.config$
+      .pipe(
+        first(),
```

Comment:
We don't need to unsubscribe because of `first()` right?

Reply:
Right, `first` ensures this observable (or actually this pipeline of it) completes on first element (which on BehaviorSubject and ReplaySubject is emitted immediatelly), and a completing observable also unsubscribe any subscription.